idurar/idurar-erp-crm
# Implement Password Strength Meter on Register Page

## Description

This pull request adds new feature which is the password strength meter on the register page. The following changes have been made:
- Implemented an accurate password strength calculation algorithm based on various criteria such as the presence of uppercase letters, lowercase letters, numbers, and special characters.
- Added a dynamic password strength meter that adjusts its color based on the strength of the password.
- Improved accessibility by adjusting the color scheme of the password strength meter trail to the same colors used previously in the app

## Related Issues

There are no related issues to this feature

## Steps to Test

1. Navigate to the register page (if you re inside the admin account you should logout)
2. Try to write something in the password field

## Screenshots

Here is a demo gif of the feature:
![demo](https://github.com/idurar/idurar-erp-crm/assets/12747343/7116d715-a685-4b56-b2ee-0026d59d3efd)

Here is a screenshot before adding the pull request:
![image](https://github.com/idurar/idurar-erp-crm/assets/12747343/52dfb86b-755b-4bbe-88e3-7fd422ad5a12)


## Checklist
